src/open_inwoner/accounts/backends.py

```diff
@@ -314,3 +314,183 @@ def update_user(self, user: AbstractUser, claims: JSONObject):
+
+        # Validate that at least one identifying claim is present (excluding pseudo_id)
+        identifying_claims = {k: v for k, v in claim_values.items() if k != "pseudo_id"}
+
+        if not any(identifying_claims.values()):
+            logger.error(
+                "eIDAS claims did not contain any identifying information",
+                claim_values=claim_values,
+                available_claim_keys=list(claims.keys()),
+            )
+
+            raise SuspiciousOperation(
+                "eIDAS claims did not contain any identifying information. "
+                "Expected at least one of: BSN, company name + legal entity ID, or first/family name. "
+                f"Available claim keys: {', '.join(claims.keys())}. "
+                f"Check the claim path configuration in the admin."
+            )
+
+        base_create_kwargs = {
+            "eidas_pseudo_id": claim_values["pseudo_id"],
+            "email": generate_email_from_string(
+                claim_values["pseudo_id"], domain="localhost"
+            ),
+        }
+
+        # Try company first, as the required fields are unambiguous
+        if claim_values["company_name"] and claim_values["legal_entity_id"]:
+            return base_create_kwargs | {
+                "login_type": LoginTypeChoices.eidas_company,
+                "company_name": claim_values["company_name"],
+                "eidas_company_id": claim_values["legal_entity_id"],
+            }
+
+        # Assume this is a person
+        person_kwargs = base_create_kwargs | {
+            "login_type": LoginTypeChoices.eidas_person_pseudo_id,
+            "first_name": claim_values["first_name"] or "",
+            "last_name": claim_values["family_name"] or "",
+        }
+
+        # Do we have a BSN? If so, add it and update login type to reflect this
+        if claim_values["bsn"]:
+            person_kwargs = person_kwargs | {
+                "bsn": claim_values["bsn"],
+                "login_type": LoginTypeChoices.eidas_person_bsn,
+            }
+
+        return person_kwargs
+
+    def filter_users_by_claims(self, claims):
+        claims = self._extract_eidas_claim_values(claims)
+        if not claims["pseudo_id"]:
+            return User.objects.none()
+
+        return User.objects.filter(eidas_pseudo_id=claims["pseudo_id"])
+
+    def create_user(self, claims: JSONObject):
+        if not (
+            create_kwargs := self._construct_user_creation_kwargs_from_claims(claims)
+        ):
+            return None
+
+        # Create new user
+        logger.debug("Creating new eIDAS user", create_kwargs=create_kwargs)
+
+        user = User.objects.create_user(**create_kwargs)
+
+        return user
+
+    def update_user(self, user: User, claims: JSONObject):
+        if not (
+            create_kwargs := self._construct_user_creation_kwargs_from_claims(claims)
+        ):
+            return None
+
+        updated = User.objects.filter(id=user.id).update(**create_kwargs)
```

Comment:
Bypasses model validation (`clean` checks bsn-users for duplicate emails). Use `user.save()` instead.